namelists/fesom2/2.1-recom-par_tracers/namelist.config

```diff
@@ -0,0 +1,95 @@
+! This is the namelist file for model general configuration
```

Comment:
What is the consensus here: do we ship namelists? Or do we simply refer to the ones given to us by the model developers? My idea would be:

* Models that ship example namelists with their source code use those. Then it is up to the model developers or maintainers to keep those working.
* Models that do not ship example namelists with their source code, we shall provide examples that we know work in conjunction with our example run configurations.

Reply:
See also: https://github.com/esm-tools/esm_tools/discussions/669

Reply:
For FESOM namelists, we have a discussion about this some time ago with @koldunovn and other FESOM users. The conclusion was that for the `master` branch (which changes through time) the namelists should be taken from the source code, while for stable branches/tags (`2.0`, `2.1`, ...) the namelists will be taken from ESM-Tools.

I don't have an opinion on what should be the general approach, as both can trigger an error in the namelist that ends in our discussion/mail-box (the code moves on but ESM-Tools namelists don't/someone changes the namelists in the FESOM source code and what was once working in ESM-Tools is not working anymore, or even worse, it works but the physics are different)

For the particular case of the par_tracers I agree with @khimstedt, that they can be taken from the source code as very advance users are using this development branch, so I have adjusted that accordingly in 8278f0b8